TombEngine/Game/control/control.cpp

```diff
@@ -145,6 +147,8 @@ GameStatus ControlPhase(int numFrames)
 
 	for (framesCount += numFrames; framesCount > 0; framesCount -= 2)
 	{
+		ClearScreenSprites();
```

Comment:
Why this call is here? Clearing of screen sprites should happen around cleaning up all other renderer objects, such as strings and particles.